clients/privacy-center/components/consent/ConsentItem.tsx

```diff
@@ -12,32 +12,39 @@ import {
   ExternalLinkIcon,
```

Comment:
Refactored this component to be more generic so that both ConfigDrivenConsent and NoticeDrivenConsent can use it